hw/ip/otbn/doc/_index.md

```diff
@@ -280,7 +282,9 @@ All RW CSRs are set to 0 when OTBN starts (when 1 is written to {{< regref "CMD.
       <td>R</td>
       <td>
         <strong>URND</strong>.
-        A random number.
+        A random number without guaranteed secrecy properties or specific statistical properties.
```

Comment:
This could be a bit stronger.  These numbers coming in here are generally DRG.3 quality.     Though I guess you are talking about the requirements in this case?

Reply:
The numbers coming into OTBN from the EDN are DRG.3, yes. But these numbers are only used as *seed* for a LFSR, and the output doesn't have DRG.3 qualities any more (https://github.com/lowRISC/opentitan/blob/master/hw/ip/otbn/rtl/otbn_rnd.sv for the implementation).

@vrozic has been working on a replacement for the LFSR which can guarantee statistical properties (make the tests pass), but we haven't implemented that yet.

@martin-lueker does that clarify things, or am I misunderstanding your comment?

Reply:
I understand the sentiment, I just worry that it could be misinterpret as referring to the actual quality of the EDN outputs. (Maybe I'm being overly sensitive here? ;) )
How about?
```suggestion
        A random number, with no requirements for special secrecy or statistical properties.
```

Reply:
Hm, I'm still confused (I'm sorry if I'm annoying you): The section where this text appears is describing URND (which carries the LFSR output) to a software developer: they want to know what kind of randomness they get when reading the register. So they care about guarantees ("What properties do the numbers read from URND have?"), really. We don't explicitly mention here that we get the seed from EDN, since that's really an implementation detail at this point.

https://docs.opentitan.org/hw/ip/otbn/doc/index.html#control-and-status-registers-csrs is the rendered doc currently.

Reply:
Oh okay fair enough then.

Reply:
Let me talk it through with Rupert tomorrow if we can find some middle ground :smile:

Reply:
After talking to Rupert I added a sentence "Use RND for high-quality randomness." to make it clear that there is actually a way to get "good" entropy from the EDN, not only the low-quality randomness the LFSR provides. I hope this goes in the right direction.